Yosemite/Yosemite/Actions/PaymentGatewayAccountAction.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+import Networking
+
+/// Defines all of the `actions` supported by the `PaymentGatewayAccountStore`.
+///
+public enum PaymentGatewayAccountAction: Action {
+    /// Retrieves and stores payment gateway accounts for the provided `siteID`
+    ///
+    case loadAccounts(siteID: Int64, onCompletion: (Result<[PaymentGatewayAccount], Error>) -> Void)
```

Comment:
Do we really need to pass the accounts in the completion block? As far as I know, all we need for now is to be able to tell if a store is enrolled into the beta, and later, if it has WCPay installed. Those could be the actual actions?

Reply:
Also, in theory, once your store updates the core data store with a new value, the ResultsController in your UI should be notified of the update

Reply:
Good idea. Changed it to simply `Result<Void, Error>`